.circleci/conditional_config.yml

```diff
@@ -460,6 +461,7 @@ jobs:
     steps:
       - attach_workspace:
          at: .
+      - run: *install_dependencies
```

Comment:
I guess we didn't really think about this angle when we switched to `.gem`s in git. You would need to remove the [`--local`](https://app.circleci.com/pipelines/github/openSUSE/open-build-service/7720/workflows/70fd1159-d1e0-4909-934a-796b18dce517/jobs/92677)

Reply:
I decided to go with caching the `.gem` files in the PR. I have also updated the [next_rails](https://github.com/openSUSE/open-build-service/wiki/next_rails/_compare/83ba6255e78d7c7adb43f41d1be1430661351c0d...0dca1095d7188cf2583db2471aec60f1398b0e0f) developer documentation to not forget this step in the future.